cypress/utils/dashboards/datasource-management-dashboards-plugin/commands.js

```diff
@@ -29,6 +29,29 @@ Cypress.Commands.add('deleteAllDataSources', () => {
   );
 });
 
+Cypress.Commands.add('createDataSourceNoAuth', () => {
+  cy.request({
+    method: 'POST',
+    url: `${Cypress.config('baseUrl')}/api/saved_objects/data-source`,
+    headers: {
+      'osd-xsrf': true,
+    },
+    body: {
+      attributes: {
+        title: `9201`,
+        endpoint: `http://localhost:9201`,
```

Comment:
to @kavilla 's point, I think it makes sense to define a var in cypress.json with default to `http://localhost:9201`, and pull from there. In case during local development, developers don't necessarily need to have an OpenSearch running exactly at 9201, they can easily pass in any domain endpoint they already have

Reply:
Good point, @wanglam mentioned the same, so added both the URL and credential for instance with security as instance in cypress.env and use that env variable in the helper function. [46d3442](https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/1146/commits/46d3442825d45ef318d72cd3a45aac3f4c0078da)